static/skywire-manager-src/src/assets/i18n/de.json

```diff
@@ -174,54 +252,35 @@
       "title": "Terminal",
       "input-start": "Skywire Terminal für {{address}}",
       "error": "Bei der Ausführung des Befehls ist ein Fehler aufgetreten."
-    },
-    "update": {
-      "title": "Update",
-      "processing": "Suche nach Updates...",
-      "processing-button": "Bitte warten",
-      "no-update": "Kein Update vorhanden.<br>Installierte Version: {{ version }}.",
-      "update-available": "Es ist ein Update möglich.<br>Installierte Version: {{ currentVersion }} <br>Neue Version: {{ newVersion }}.",
-      "done": "Ein Update für den Visor wird installiert.",
-      "update-error": "Update konnte nicht installiert werden.",
-      "install": "Update installieren"
     }
   },
+  
+  "update": {
+    "title": "Update",
```

Comment:
I am not 100% sure where these are used. But should we not use the german words here?

Reply:
I think so, changed to 'Aktualisierung'